frontend/src/components/HomePage.js

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import { Box, Button, Typography } from "@mui/material";
+import logo from "../images/logo.svg";
+import { useNavigate } from "react-router-dom";
+import "../css/HomePage.scss"
+import { styled } from '@mui/system';
+
+const StyledButton = styled(Button)`
+    margin-top: 3rem;
+    width: 320px;
+    background-color: #ffcb36;
+    color: #0e141f;
+    font-size: 1.5rem;
+`
+
+export const HomePage = () => {
+    const navigate = useNavigate();
+
+    const uniqueid = () => {
+        // always start with a letter (for DOM friendlyness)
+        var idstr = String.fromCharCode(Math.floor(Math.random() * 25 + 65));
+        do {
+            // between numbers and characters (48 is 0 and 90 is Z (42-48 = 90)
+            var ascicode = Math.floor(Math.random() * 42 + 48);
+            if (ascicode < 58 || ascicode > 64) {
+            // exclude all chars between : (58) and @ (64)
+            idstr += String.fromCharCode(ascicode);
+            }
+        } while (idstr.length < 32);
+
+        return idstr;
+    }
+
+    const createRoom = () => {
+        let uniqueRoomId = "/game/" + uniqueid();
+        navigate(uniqueRoomId);
+    }
+
+    return (
+        <>
+        <Box className="hero is-large">
+            <Box className="illustration-container">
+                <Box className="section container">
+                    <Box className="brand">
+                        <img src={logo} alt="Tambola" />
+                    </Box>
+                </Box>
+                <Box className="container">
+                    <Box className="hero-text-container">
+                    <Typography variant="h1" className="title largest-text">
+                        Tambola
+                    </Typography>
+                    <Typography variant="h2" className="subtitle is-size-1">
+                        Anywhere Anytime
+                    </Typography>
+                    <Typography variant="p" className="is-size-3 sub-sub-heading">
+                        Same Tambola fun with your loved ones on your browser. Create a
+                        game and then share link to play together.
+                    </Typography>
+                    <StyledButton className="pulse" onClick={createRoom}>
+                        Create Game
+                    </StyledButton>
+                    </Box>
+                </Box>
+            </Box>
+        </Box>
+        <Box className="container"><hr /></Box>
+        </>
+    );
+};
+
+export default HomePage;
+
+
+
+
+
+
+
+// <!DOCTYPE html>
+// <html>
+//   <head>
+//     <meta
```

Comment:
We also need these meta tags for SEO. So might have to research that `react-router` does not mess up our SEO.